databricks/koalas/frame.py

```diff
@@ -5538,6 +5562,23 @@ def replace(
         2     Thor  Stormbuster
         3     Hulk        Smash
 
+        You can also use an iterable object that returns a list of columns,
+        such as `tuple` or `list`, as input to the `subset` parameter.
+
+        >>> df.replace('Mjolnir', 'Stormbuster', subset=('weapon',))
+              name       weapon
+        0   Rescue      Mark-45
+        1  Hawkeye       Shield
+        2     Thor  Stormbuster
+        3     Hulk        Smash
+
+        >>> df.replace('Mjolnir', 'Stormbuster', subset=['weapon'])
```

Comment:
Let's move one of both to the test cases instead of doctest. Seems a bit verbose to have two examples to explain the iterable object as an argument.

Reply:
```python
>>> kdf.replace('Mjolnir', 'Stormbuster', subset=('weapon',))
                     name       weapon
0.342778          Ironman      Mark-45
0.087444  Captain America       Shield
0.179212             Thor  Stormbuster
0.522174             Hulk        Smash

>>> pdf.replace('Mjolnir', 'Stormbuster', subset=('weapon',))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: replace() got an unexpected keyword argument 'subset'
```

@HyukjinKwon 
When I tried adding the above test to the test case, I found that pandas does not support the `subset` parameter. So when I looked into the old version, pandas didn't support the `subset` parameter from the beginning. I found that the current `replace` parameter matches the spark `replace`. So, what are your thoughts on deleting a `subset` from Koalas for pandas?

ref>

[pandas 0.9.0 replace()](https://pandas.pydata.org/pandas-docs/version/0.9.0/generated/pandas.DataFrame.replace.html?highlight=replace#pandas.DataFrame.replace)

[pandas 1.0.1 replace()](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.replace.html)

[pyspark 2.1.3 replace()](https://spark.apache.org/docs/2.1.3/api/python/pyspark.sql.html#pyspark.sql.DataFrame.replace)